### DISCUSSION TITLE
RPI4 Seeed Mic 2 configuration issue.

### DISCUSSION THREAD

**Discussion by Go-rak** [mcguirepr89/BirdNET-Pi#706] [Q&A] [answered]
Hello,
I've tried to setup a seeed mic 2 on my Rpi 4, I followed this post #230

But I have this issue when I check my journalctl -f I've got those issues :

Jan 20 23:55:48 birdnetpi birdnet_recording.sh[8135]: + arecord -f S16_LE -c2 -r48000 -t wav --max-file-time 15 -D 3 --use-strftime /home/pi/BirdSongs/%B-%Y/%d-%A/%F-birdnet-%H:%M:%S.wav
Jan 20 23:55:48 birdnetpi birdnet_recording.sh[8141]: ALSA lib pcm.c:2660:(snd_pcm_open_noupdate) Unknown PCM 3
Jan 20 23:55:48 birdnetpi birdnet_recording.sh[8141]: arecord: main:830: audio open error: No such file or directory
Jan 20 23:55:48 birdnetpi systemd[1]: birdnet_recording.service: Main process exited, code=exited, status=1/FAILURE
Jan 20 23:55:48 birdnetpi systemd[1]: birdnet_recording.service: Failed with result 'exit-code'.

It seems my card is not well configured, I tried to test mic and sound by this command :
arecord -D "plughw:3,0" -f S16_LE -r 16000 -d 5 -t wav test.wav
aplay -D "plughw:3,0" test.wav

And it works well.

Have you any clues to help me to setup this configuration ?
Thank you.

**Comment by ehpersonal38** [comment]
`3` as the audio card is not right, you need to use the full name like you did when manually calling arecord, `plughw:3,0`, like so:
![chrome_584piT4MGi](https://user-images.githubusercontent.com/103586016/213822784-116572c5-35c7-4e81-835e-8e0189906948.png)

**Comment by Go-rak** [comment]
I tried it, it doesn't work as I can't add the quote :  "plughw:3,0"
but it seems a good way I will work on it.

**Comment by Go-rak** [comment, marked as answer]
Hello,
It works with :
hw:3,0
Thank for your help !
